@xen-orchestra/backups/_VmBackup.js

```diff
@@ -324,6 +326,8 @@ exports.VmBackup = class VmBackup {
   }
 
   async run() {
+    await asyncMap(this._writers, writer => writer.beforeBackup(this.vm.uuid))
```

Comment:
Not necessary, writers have access to this instance:

```suggestion
    await asyncMap(this._writers, writer => writer.beforeBackup())
```